eng/Version.Details.xml

```diff
@@ -297,22 +297,22 @@
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>03c21467c97e75103d429a0e67504e402f6da17d</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="7.0.0-beta.22419.1">
+    <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="7.0.0-beta.22423.1">
```

Comment:
@JamesNK note I'm letting the Arcade update come through. Some Good Stuff:tm: in the latest update.